polaris/curation/viz_utils.py

```diff
@@ -52,23 +67,46 @@ def verify_stereoisomers(data_cols: List[str], dataset: PandasDataFrame):
 
     """
     figs = []
+    if legend_cols is None:
+        legend_cols = []
     for col in data_cols:
         logger.info(f"Verify the stereo ismomers for readout `{col}`")
         cliff_col = f"CLASS_{col}_stereo_cliff"
         if cliff_col not in dataset:
-            raise ValueError(
-                "The stereo chemistry information is unavailable. \
-                             Please run the <polaris.curation.run_chemistry_curation> first."
-            )
+            cliff_col = f"{col}_stereo_cliff"
```

Comment:
Turn into a setting, e.g. `ACTIVITY_CLIFF_COLL_SUFFIX`? Should be configurable through a Pydantic `BaseSettings` class.